test/unit/messages_test.rb

```diff
@@ -0,0 +1,7 @@
+require File.join(File.dirname(__FILE__), 'test_helper')
+require 'hammer_cli/testing/messages'
+
+include HammerCLI::Testing::Messages
+describe "message formatting" do
+  check_commands(HammerCLI::MainCommand)
```

Comment:
@tstrachota while we are including it in a global space I'd prefer more specific  name. Also for test readability it should suggest what we are testing. How about `check_command_messages`?